### ISSUE TITLE
badge: <Estonian> for <Le>

### ISSUE THREAD

**Issue by Lefromrevolt** [revoltchat/translations#155]
### What language(s) did you translate?

Estonian

### What is your Revolt ID?

01GVK6D8A5S722BVNEP35263T2

### Link to Weblate profile

https://weblate.insrt.uk/user/Le/

### Validations

- [X] The language has a maintainer (see the full list [here](https://github.com/revoltchat/translations#languages)).
- [X] You agree that your translations are to a reasonable standard.

**Comment by Rexogamer** [comment]
cc @Asl-script

**Comment by Asl-script** [comment]
You have translated 26 total strings. I looked over your translations and I only found one issue. 

You translated “This user is not old enough to have an account” to “See kasutaja pole konto jaoks piisavalt vana” which means “This user isn’t old enough **for** and account.

This issue is extremely small and I will fix it. Thanks for your contributions, I approve! 😊

**Comment by Asl-script** [comment]
You have translated 26 total strings from tonight. I looked over your translations and I only found one issue.

Some Notes:
You translated “This user is not old enough to have an account” to “See kasutaja pole konto jaoks piisavalt vana” which means “This user isn’t old enough for and account”.

Conclusion:
This issue is extremely small and I will fix it. Everything else seems fine to me. Thanks for your contributions, I approve! 😊
